corehq/apps/auditcare/urls.py

```diff
@@ -1,30 +1,8 @@
-from django.conf import settings
 from django.conf.urls import url
 
-from .utils import login_template, logout_template
-from .views import (
-    audited_login,
-    audited_logout,
-    audited_views,
-    export_all,
-)
+from .views import audited_views, export_all
 
 urlpatterns = [
     url(r'^auditor/export/$', export_all, name='export_all_audits'),
     url(r'^auditor/views/$', audited_views, name='audit_views'),
-
-    # directly overriding due to wrapped functions causing serious problems with tests
-    url(r'^accounts/login/$', audited_login, {'template_name': login_template()}, name='auth_login'),
-    url(r'^accounts/logout/$', audited_logout, {'template_name': logout_template()}, name='auth_logout'),
```

Comment:
As far as I could tell these views (which have actual paths of `/auditcare/accounts/login` and `/auditcare/accounts/logout` because of where they are installed in the Django URL structure) are not used.

@czue do you know how these were originally intended to be used?

Reply:
I have no memory, but my intuition is that they were intended to override the base `accounts/login` (either in HQ or some other app) and just never worked that way in HQ or haven't worked that way in a long long time.

As a point of history it might be worth knowing that `auditcare` was first written for a standalone project that was separate for HQ and then incorporated into HQ, so stuff like this that "never worked" seems likely to have slipped through.